src/components/DappBrowser/control-panel/ControlPanel.tsx

```diff
@@ -140,7 +139,7 @@ export const ControlPanel = () => {
     const bluetoothWallets: ControlPanelMenuItemProps[] = [];
     const readOnlyWallets: ControlPanelMenuItemProps[] = [];
 
-    const accountBalances: Record<string, number> = {};
+    const accountBalances: Record<string, string | number> = {};
```

Comment:
nit: `totalBalanceAmount` will always be a string as it is a result of `add` which is always a string (will leave 
a similar comment in `useWalletBalances`)